README.md

```diff
@@ -107,31 +107,26 @@ Assuming the repo is in your home directory
 ```
 cd ~/usb-sd-creator
 
-./osx_build.sh
-```
-
-If building again cleanup using:
-```
-./osx_clean.sh
+cmake -S . -B build -D CMAKE_PREFIX_PATH="/Users/$USER/Qt/6.6.2-static" && cmake --build build
 ```
 
 ### 5. Run USB-SD-Creator
 
 #### Open the app
 
-Simply double click the app from a finder window in the root of the repo: `LibreELEC USB-SD Creator`
+Simply double click the app from a finder window in the `build` inthe repo: `build/LibreELEC USB-SD Creator`
```

Comment:
Looks like an extra "inthe" crept in this line?

Reply:
Fixed, rewrote instructions using AQT to manage QT install.